transmitter.py

```diff
@@ -21,16 +35,45 @@ def modulate(msg):
 	return sendarr
 
 def play_audio(filename):
+	''' 
+	The audio is played using pyaudio.Pyaudio.open() where the wave file of the audio
+	is input as read-binary. 
+
+	:param .wav filename: audio file
+	:wave audio file is played as output	
+	'''
+
 	print("Transmitting...")
 	chunk = 1024  
 
 	f = wave.open(filename,"rb")  
+	# instantiation of Pyaudio
+	# this sets up the  portaudio system
 	p = pyaudio.PyAudio()  
+
+	# stream is opened
 	stream = p.open(format = p.get_format_from_width(f.getsampwidth()),  
 	                channels = f.getnchannels(),  
 	                rate = f.getframerate(),  
-	                output = True)  
-	data = f.readframes(chunk)  
+	                output = True)
+	''' 
+	:param int format: sampling size and format
```

Comment:
This is not a function definition.

Reply:
You want me to remove this multi-line comment?

Reply:
Yes. I think this reduces readability.